### DISCUSSION TITLE
HTU21 not working with a second sensor

### DISCUSSION THREAD

**Discussion by DiaconuFlorinNarcis** [arendst/Tasmota#19347] [Q&A] [answered]
I have this generic esp8266 board and 2x HTU21D temp ad humidity sensor board with the bridge pins unsoldered just like the picture:
 ![board](https://s13emagst.akamaized.net/products/48304/48303789/images/res_c24471d92b5e9522b2be83a95906fbc3.jpg?width=450&height=450&hash=B142F2EEC75B190D6A2B7A40AF676A5E)
When i first wired them together i bridged the pull-up pins only for one board the second wan was unsoldered.

The first fire is giving me the following error:
on weblog 3
> 21:38:36.400 SNS: HTU21 missed 5
> 21:38:36.400 SNS: HTU21 missed 5
> 21:38:36.400 SNS: HTU21 missed 5

while this error is repeating the I2cScan results in 1 device found:
> 21:47:50.412 CMD: I2CScan
> 21:47:50.418 CMD: Grp 0, Cmd 'I2CSCAN', Idx 1, Len 0, Pld -99, Data ''
> 21:47:50.437 RSL: RESULT = {"I2CScan":"Device(s) found at 0x40"}

If i disconnect one sensor everything is working well and i see data from one sensor.

How do I make multiple sensors work on the same board?

**Comment by barbudor** [comment, marked as answer]
That is basics of I2C : It is a bus and each device on a bus must have a different address.
HTU21 cannot be configured for different address. 0x40 is hardcoded in the chip and can't be changed.
So only 1 HTU21 per ESP8266

BME280 supports addresses 0x76 and 0x77
So you could have 1x HTU + 2x BME280 on the same tasmota
